acsylla/_cython/result/value.pyx

```diff
@@ -26,6 +26,26 @@ cdef class Value:
 
         return output
 
+    def timeuuid(self):
+        cdef CassError error
+        cdef CassUuid uuid
+
+        error = cass_value_get_uuid(self.cass_value, &uuid)
+        raise_if_error(error)
+
+        return cass_uuid_timestamp(uuid)
```

Comment:
I have my objections providing this logic in the driver, I know that Cassandra provides them as primitive methods for building UUIDs based on protocol V1 but having the feeling that the Driver went to far away on what responsabilities would need to be done by the driver, and what others would need to be done by the caller.

Python already provides a UUID1 generation [1]. So, unless I'm missing something, I would prefer to ask the user for using the one provided by Python, the output can be used normally as a simple UUID value compatible with Cassandra/Scylla - with the other part of the PR that you are providing.

WDYT?

[1] https://docs.python.org/3/library/uuid.html#uuid.uuid1

Reply:
IMHO i found useful the uuid utils that cassandra driver provides. `uuid` module in  python doesnt have any method to convert uuid to timestamp so we will need to code ourselves, i found useful having exactly the same logic that the server is using to calculate the uuids in the client as well. But i dont have any hard ideas, so if you find that task better done in the client bussiness logic ill remove the utility methods :)

Reply:
I would prefer for now to not implement - provide - this interface to the user.

Reply:
👍 ill remove. just for reference, this will be the logic to mimic those calls: https://github.com/samuraisam/time_uuid/blob/master/time_uuid/__init__.py

Reply:
done!